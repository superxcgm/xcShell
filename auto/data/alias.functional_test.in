mkdir alias_dir
cd alias_dir
touch hello
mkdir subdir
alias
alias ls='ls -G'
ls
alias ll='ls -lh'
ll
alias
alias ls
