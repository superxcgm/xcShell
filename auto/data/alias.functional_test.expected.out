xcShell_functional_test > mkdir alias_dir
xcShell_functional_test > cd alias_dir
alias_dir > touch hello
alias_dir > mkdir subdir
alias_dir > alias
alias_dir > alias ls='ls -G'
alias_dir > ls
hello
subdir
alias_dir > alias ll='ls -lh'
alias_dir > alias cdsub='cd subdir'
alias_dir > cdsub
subdir > alias
cdsub='cd subdir'
ll='ls -lh'
ls='ls -G'
subdir > alias ls
ls='ls -G'
subdir > 